.gitignore

```diff
@@ -1,3 +1,3 @@
 .idea/
 coverage/
-public/
\ No newline at end of file
+public/uploads/*
```

Comment:
Hmmmm - I thought we were storing this in the repo - but this line would prevent this